bpf/bpf_netdev.c

```diff
@@ -140,7 +141,14 @@ static inline int handle_ipv6(struct __sk_buff *skb)
 #endif
 
 	BPF_V6(node_ip, ROUTER_IP);
+
 	flowlabel = derive_sec_ctx(skb, &node_ip, ip6);
+#ifdef FROM_HOST
+	/* For packets from the host, the identity can be specified via skb->mark */
+	if (skb->mark) {
+		flowlabel = skb->mark;
+	}
+#endif
```

Comment:
Yes, it will initialize the flowlabel with `FIXED_SRC_SECCTX` which is set to `HOST_ID` in the `FROM_HOST`'s case. This means that in case no `skb->mark`, packets will look like they come from the host from a security perspective.